contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,105 @@
+
+
+class PollHandler(object):
+    '''
+    This plugin facilitates polling.
+    '''
+
+    def __init__(self):
+        # polls is a dict of dict. First dict corresponds to poll title,
+        # second refers to answers in poll
+        self.polls = dict()
+
+        # streams is a convenience object that keeps track of the latest
+        # question for each stream.
+        self.streams = dict()
```

Comment:
What if users create multiple polls? How do we handle them?

Also, is there a way to "close" a poll, meaning no users can respond to it anymore?